featuretools/tests/synthesis/test_dask_dfs.py

```diff
@@ -21,25 +22,26 @@ def test_single_table_dask_entityset():
                                    ""]})
     values_dd = dd.from_pandas(df, npartitions=2)
     vtypes = {
-        "id": ft.variable_types.Id,
-        "values": ft.variable_types.Numeric,
-        "dates": ft.variable_types.Datetime,
-        "strings": ft.variable_types.NaturalLanguage
+        "values": ww.logical_types.Double,
+        "dates": ww.logical_types.Datetime,
+        "strings": ww.logical_types.NaturalLanguage
     }
-    dask_es.entity_from_dataframe(entity_id="data",
-                                  dataframe=values_dd,
-                                  index="id",
-                                  variable_types=vtypes)
+    dask_es.add_dataframe(
+        dataframe_name="data",
+        dataframe=values_dd,
+        index="id",
+        logical_types=vtypes)
```

Comment:
Might make sense to rename vtypes to ltypes here

Reply:
renamed